File: hive/src/test/java/com/dbx/agent/hive/HiveAgentTest.java
package com.dbx.agent.hive;

import org.junit.jupiter.api.Test;

class HiveAgentTest {
    @Test
    void hiveJdbcStandaloneRuntimeClassesAreAvailable() throws ClassNotFoundException {
        Class.forName("org.apache.hive.jdbc.HiveDriver");
        Class.forName("org.apache.hive.org.apache.thrift.protocol.TProtocol");
    }
}

File: hive/src/test/kotlin/com/dbx/agent/hive/HiveAgentTest.kt
package com.dbx.agent.hive

import kotlin.test.Test

class HiveAgentTest {
    @Test
    fun hiveJdbcStandaloneRuntimeClassesAreAvailable() {
        Class.forName("org.apache.hive.jdbc.HiveDriver")
        Class.forName("org.apache.hive.org.apache.thrift.protocol.TProtocol")
    }
}
